.github/workflows/action.yml

```diff
@@ -0,0 +1,33 @@
+name: Unit test and Publish
+on:
+  push:
+    branches: '**'
+  pull_request:
+    branches: [ "develop" ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.10
```

Comment:
Is this suppose to be 3.9 considering the python-version a couple of lines down is 3.9?

Reply:
Yup, it should be 3.9. 3.10 had a bug that was failing the pypi publish.